cmd/opm/alpha/diff/cmd.go

```diff
@@ -0,0 +1,101 @@
+package diff
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/internal/action"
+	"github.com/operator-framework/operator-registry/internal/declcfg"
+	imgreg "github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+	"github.com/operator-framework/operator-registry/pkg/lib/certs"
+)
+
+const (
+	retryInterval = time.Second * 5
+	timeout       = time.Minute * 1
+)
+
+type diff struct {
+	oldRefs []string
+	newRefs []string
+
+	caFile   string
+	pullTool string
```

Comment:
Any reason we need `pullTool` to be anything other than `none`? Or can we just hardcode use of containerd to pull images?

Reply:
Nope, this was copied from an earlier version of `render` I think.